### PR TITLE
JCES-1336: Explain how Atlassians get AWS creds

### PR DESCRIPTION
**Pull request by dagguh**

### CODE REVIEW

File: docs/AWS-CREDENTIALS.md [hunk 1]
```diff
@@ -3,5 +3,13 @@
...
+## AWS credentials for Atlassian engineering
+If you are a member of an Atlassian engineering departament/team,
+you can obtain AWS credentials via our internal tool - [cloudtoken].
+You can run it in daemon mode: `cloudtoken -d`
+and assume the `arn:aws:iam::695067801333:role/Jira-Server-Perf-Dev-Protection-Pipeline` role.
+It already has the permissions set up.
+
 [full JPT permissions]: https://bitbucket.org/atlassian/jira-performance-tests/src/master/aws-policy.json
 [`AwsVus`]: ../src/test/kotlin/jces1209/AwsVus.kt
+[cloudtoken]: http://go.atlassian.com/cloudtoken
```
Comment (Mac-Ste) [inline docs/AWS-CREDENTIALS.md:15]: maybe let's add there `go/aws` as well?

**Review by Mac-Ste — APPROVED** [review]

File: docs/AWS-CREDENTIALS.md (hunk 1 above)
Comment (dagguh) [reply docs/AWS-CREDENTIALS.md:15]: What's the use case for `go/aws`?

Comment (Mac-Ste) [reply docs/AWS-CREDENTIALS.md:15]: I thought about quickly checking roles that you can assume, like before setting up `cloudtoken`

Comment (dagguh) [reply docs/AWS-CREDENTIALS.md:15]: We need `cloudtoken` anyway, and it will list the roles. I want to avoid inflating the process.

**Pull request merged**